src/schema/schema_alter.c

```diff
@@ -237,8 +269,7 @@ __schema_alter(WT_SESSION_IMPL *session, const char *uri, const char *newcfg[])
     if (WT_PREFIX_MATCH(uri, "table:"))
         return (__alter_table(session, uri, newcfg, exclusive_refreshed));
     if (WT_PREFIX_MATCH(uri, "tiered:"))
-        return (__wt_schema_tiered_worker(session, uri, __alter_file, NULL, newcfg, flags));
-
+        return __alter_tiered(session, uri, newcfg, flags);
```

Comment:
Parenthesis around the value to be returned.

Reply:
Done